app.py

```diff
@@ -9,6 +10,26 @@
 from urllib3.exceptions import SSLError
 from collections import OrderedDict
 from typing import Callable
+from fast_autocomplete import AutoComplete
+
+AUTOCOMPLETE_MAX_COST = 3
+AUTOCOMPLETE_SIZE = 5
+
+#codes inspired -> JKL project owner TOM ..
+
+words = {}
+
+def getAutocompleteEntries() -> dict :
+    words = {}
+    with gzip.open('../wikidump/enwiki-20210820-abstract.xml.gz', 'rb') as f:
+         doc_id = 1
+         for _, element in etree.iterparse(f, events=('end',), tag='doc'):
+             title = element.findtext('./title')
+             words[title[11:]]={}
+             doc_id += 1
```

Comment:
**[Required]** This `doc_id += 1` can also be removed since it is unused

Reply:
ok